src/styles/main.scss

```diff
@@ -1,3 +1,48 @@
-body {
-  margin: 0;
+@import "variables";
+@import "mixins";
+@import "page";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $length * 7 + $gap * 6;
+  margin: auto;
+  padding: 10px;
+  gap: 1px;
```

Comment:
I think you missed this one, you can use the variable

Reply:
Yep, I'll be on it